mycore-mods/src/main/java/org/mycore/mods/MCRMODSWrapper.java

```diff
@@ -343,8 +343,8 @@ private List<Element> getCategoryNodes() {
     }
 
     private static String xPathRelationshipTypeTest() {
-        return Stream.of(MCRMODSRelationshipType.values())
+        return "(" +Stream.of(MCRMODSRelationshipType.values())
```

Comment:
space after "+", did you format the code?

Reply:
i removed it, because it was only a temp patch until 2022 gets merged to 2023.